types/whatwg-url/index.d.ts

```diff
@@ -1,23 +1,21 @@
-// Type definitions for whatwg-url 8.2
+// Type definitions for whatwg-url 11.0
 // Project: https://github.com/jsdom/whatwg-url#readme
 // Definitions by: Alexander Marks <https://github.com/aomarks>
 //                 ExE Boss <https://github.com/ExE-Boss>
+//                 BendingBender <https://github.com/BendingBender>
```

Comment:
Just to verify, are you wanting to become an owner? https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners

If you're just looking for authorship credit, that's handled by the git history; this block is explicitly for code review approval purposes and you'll get pinged on every PR to this package.

Reply:
Yes, that's what I wanted.